eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -386,8 +387,14 @@ function generateRetryTestsHelmValues ($pkg, $releaseName, $generatedHelmValues)
     $podOutput = RunOrExitOnFailure kubectl get pods -n $pkg.namespace -o json
     $pods = $podOutput | ConvertFrom-Json
 
+    # helm version example: v3.11.2+g912ebc1
```

Comment:
Let's do this check in the `CheckDependencies` function rather than in the retry function. I think it's ok to require the latest version even if it's only used for a subset of features that not everyone will touch.

Reply:
Yeah that was my concern, is to not affect other's who are not using this feature But I guess it's fine to encourage people to use the latest tools

Reply:
Yeah otherwise you could get halfway through an installation and then fail out. Better to alert up front if anything is missing. One option we could consider is using `CheckDependencies` but checking if `-RetryFailedTests` is set, that way you fail fast but still only if you're using the feature.